### ISSUE TITLE
allow junos_install_config to do "load update" besides "load overwrite"

### ISSUE THREAD

**Issue by henrchen** [Juniper/ansible-junos-stdlib#207]
Currently we can use merge, overewrite and replace.

But Junos also has a "update" switch which does almost the same thing as "overwrite", but it does not re-parse any unchanged statements.

This is helpful if we want to periodically audit our devices to make sure no unauthorized (i.e. not approved, not vision controlled) changes is made. An "overwrite" re-parse all the statements even there is no change. This is a little bit too intrusive, i.e. reset BGP table age and re-learn all the routes.

**Comment by vnitinv** [comment]
@henrchen We will get this feature added.

**Comment by stacywsmith** [comment]
This feature has been implemented by #218 and Juniper/py-junos-eznc#681.

Please test by installing the latest code for both PyEZ and the Junos Ansible modules from GitHub.

You can install latest PyEZ from GitHub with:
```
sudo pip install --upgrade git+https://github.com/Juniper/py-junos-eznc.git
```

You can install latest Junos Ansible modules from GitHub with:
```
sudo ansible-galaxy remove Juniper.junos
sudo ansible-galaxy install git+https://github.com/Juniper/ansible-junos-stdlib.git,,Juniper.junos
```

Please let us know if you experience any problems with the feature.
